tests/TEMP_README.txt

```diff
@@ -0,0 +1,18 @@
+Tests for codalab are written in Selenium Driver
```

Comment:
Oh. I just called it temp because I thought that we might move these instructions to some other document.